frontend/public/locales/en/translation.json

```
@@ -3069,6 +3079,7 @@
   "Version": "Version",
   "Version availability": "Version availability",
   "Version compatibility": "Version compatibility",
+  "Versions is a list of non-empty strings that specifies which installed versions are compliant when set to 'inform' mode, and which installPlans are approved when when set to 'enforce' mode\": \"Versions is a list of non-empty strings that specifies which installed versions are compliant when set to 'inform' mode, and which installPlans are approved when when set to 'enforce' mode": "Versions is a list of non-empty strings that specifies which installed versions are compliant when set to 'inform' mode, and which installPlans are approved when when set to 'enforce' mode\": \"Versions is a list of non-empty strings that specifies which installed versions are compliant when set to 'inform' mode, and which installPlans are approved when when set to 'enforce' mode",
```

It seems like you are attempting to monospace `like this`, but single quotes were used. I'm just not sure how this is rendered from the console
```suggestion
  "Versions is a list of non-empty strings that specifies which installed versions are compliant when set to `inform` mode, and which `installPlans` are approved when set to `enforce` mode.\": \"Versions is a list of non-empty strings that specifies which installed versions are compliant when set to `inform` mode, and which `installPlans` are approved when set to `enforce` mode.": "Versions is a list of non-empty strings that specifies which installed versions are compliant when set to `inform` mode, and which `installPlans` are approved when set to `enforce` mode".\": \"Versions is a list of non-empty strings that specifies which installed versions are compliant when set to `inform` mode, and which installPlans are approved when when set to `enforce` mode.",
```

---

let me try

---

Back-ticks will not work in the console to display monospace. If you want to format text, you need to use tags in the translation string along with the `<Trans>` component. For cases like this where the text is very long, I would also recommend using a symbolic key rather than the actual text.

---

Thanks for your advise!! @KevinFCormier